composer.json

```diff
@@ -54,7 +54,6 @@
       "@cs-check",
       "@test"
     ],
-    "upload-coverage": "coveralls -v",
```

Comment:
Why this change, exactly? The point of having a script is so that we can make a change to the script, without having to make a change to the travis configuration...

This should change to:

```json
"upload-coverage": "php-coveralls -v",
```

and then the `.travis.yml` only needs a change in terms of the `COVERAGE_DEPS` declaration.

I'll take care of that during merge.

Reply:
@weierophinney I think this is what we agreed on slack and what was merged into maintainers repository:
https://github.com/zendframework/maintainers/pull/38

The script is only used in travis build and imho it shouldn't be available to the developers.

Reply:
@webimpress I'd totally forgotten that conversation!

I've opened #288 to re-instate these changes.